### ISSUE TITLE
📚 Documentation: Double check queries

### ISSUE THREAD

**Issue by stnguyen90** [appwrite/website#1353]
### 💭 Description

The syntax for some queries don't seem to match between the docs and the SDKs.

For example, the Python SDK's has [`Query.and_queries()`](https://github.com/appwrite/sdk-for-python/blob/dba5c5d63e9b426753a73793bf8d734566a44585/appwrite/query.py#L107), but the docs show `Query.and()`:

![image](https://github.com/user-attachments/assets/dc678134-5763-43d8-9d85-17755bd3e152)

In addition, the Swift SDK requires the `value` label like [`Query.equal("released", value: [true])`](https://github.com/appwrite/sdk-generator/blob/2dd681f3c773b2054f0b28659cf39a1f1649edcf/tests/languages/apple/Tests.swift#L151), but the docs just show `Query.equal("title", ["Iron Man"])`:

![image](https://github.com/user-attachments/assets/d4fc833f-0c28-4575-ba9f-4cfaeb6db669)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by ChiragAgg5k** [comment]
![Image](https://github.com/user-attachments/assets/fd7a29d6-618c-4941-bc00-0b910ba8f356)

@stnguyen90 regarding your first issue, it seems like only the python sdk has this discrepancy for using `and_queries()`,  while rest all use `.and()` syntax. perhaps it would be better to fix this in python sdk itself?

**Comment by stnguyen90** [comment]
@ChiragAgg5k, yes the Python SDK is different. We can't use `and` because it's [reserved](https://docs.python.org/3/reference/lexical_analysis.html#keywords).

**Comment by ChiragAgg5k** [comment]
oh alrightt

**Comment by ChiragAgg5k** [comment]
I can work on the issue, can you assign it to me?

**Comment by stnguyen90** [comment]
@ChiragAgg5k, assigned!
